src/js/tasks-function.js

```diff
@@ -0,0 +1,223 @@
+// eslint-disable-next-line max-classes-per-file
+class Tasks {
+  constructor(description, completed = false, index) {
+    // eslint-disable-next-line no-unused-expressions
```

Comment:
- [ ] Please remove any `eslint-disable` as that defies the purpose of using linters in the first place.